libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/contact-data/dh-market-participant-actor-contact-data.component.html

```diff
@@ -88,6 +106,11 @@
       <mat-cell *matCellDef="let element">
         <ng-container *ngIf="!element.isNewPlaceholder">
           <watt-button
+            [attr.aria-label]="
+              element.isExisting
+                ? 'delete for existing contact'
```

Comment:
This label is also used by screen-readers, so maybe just remove `for` here? 
```suggestion
                ? 'delete existing contact'
```